pkg/envoy/cds.go

```diff
@@ -81,32 +80,6 @@ func (c HasClusterCheck) Run() error {
 	return nil
 }
 
-func (c HasClusterCheck) listServicesForPod() ([]*corev1.Service, error) {
-	var serviceList []*corev1.Service
-	svcList, err := c.k8s.CoreV1().Services(c.dstPod.Namespace).List(context.Background(), metav1.ListOptions{})
-	if err != nil {
-		return nil, errors.Wrapf(err, "failed to list services in namespace %s", c.dstPod.Namespace)
-	}
-
-	for _, svc := range svcList.Items {
-		svc := svc
-		if svc.Namespace != c.dstPod.Namespace {
-			continue
-		}
```

Comment:
@nojnhuh wanted to make sure it's okay that is not part of the helper being used now... the Services.List is called only on the dstPod's Namespace so this check feels a bit redundant...?

Reply:
Sorry this is so late, but yes, you're right.